types/react-native/index.d.ts

```diff
@@ -8151,6 +8152,14 @@ export interface VibrationStatic {
     cancel(): void;
 }
 
+export namespace Appearance {
+    type ColorSchemeName = 'light' | 'dark';
+
+    export function getColorScheme(): ColorSchemeName;
+    export function addListener(callback: (eventName: string) => void): void;
```

Comment:
Looks like you were looking at the `NativeAppearance` module, rather than [the public API](https://github.com/facebook/react-native/commit/63fa3f2#diff-55c2992d993407398c62bf19f803088f).

Reply:
You are right, thanks for the heads-up